src/main/java/net/atos/zac/app/informatieobjecten/converter/RESTInformatieobjectConverter.java

```diff
@@ -90,13 +93,16 @@ public RESTEnkelvoudigInformatieobject convertToREST(final ZaakInformatieobject
         return convertToREST(enkelvoudigInformatieObject, zaak);
     }
 
-    public RESTEnkelvoudigInformatieobject convertToREST(final AbstractEnkelvoudigInformatieobject enkelvoudigInformatieObject) {
+    public RESTEnkelvoudigInformatieobject convertToREST(
+            final AbstractEnkelvoudigInformatieobject enkelvoudigInformatieObject) {
         return convertToREST(enkelvoudigInformatieObject, null);
     }
 
-    public RESTEnkelvoudigInformatieobject convertToREST(final AbstractEnkelvoudigInformatieobject enkelvoudigInformatieObject, final Zaak zaak) {
-        final EnkelvoudigInformatieObjectLock lock = enkelvoudigInformatieObject.getLocked() ? enkelvoudigInformatieObjectLockService.findLock(
-                enkelvoudigInformatieObject.getUUID()) : null;
+    public RESTEnkelvoudigInformatieobject convertToREST(
+            final AbstractEnkelvoudigInformatieobject enkelvoudigInformatieObject, final Zaak zaak) {
+        final EnkelvoudigInformatieObjectLock lock = enkelvoudigInformatieObject.getLocked() ?
+                enkelvoudigInformatieObjectLockService.findLock(enkelvoudigInformatieObject.getUUID()).orElse(null)
+                : null;
```

Comment:
is de if constructie hier nog nodig? of kun je ook alleen  

enkelvoudigInformatieObjectLockService.findLock(enkelvoudigInformatieObject.getUUID()).orElse(null) 

doen?